react/components/Portal.tsx

```diff
@@ -17,6 +18,10 @@ interface Props {
 export function getContainer(container?: ContainerType) {
   container = typeof container === 'function' ? container() : container
 
+  if (!container) {
+    return undefined
```

Comment:
⛏️  How about returning `null` instead of `undefined`?

Reply:
Same thing at line 46

Reply:
I can change here but not on line 46
![image](https://user-images.githubusercontent.com/284515/104496059-3c102800-55b7-11eb-9616-82eae3721bd4.png)